specification/src/main/asciidoc/webprofile/jpa-cdi/CDI-JPA.adoc

```diff
@@ -1,9 +1,16 @@
 :cdi-spec: https://jakarta.ee/specifications/cdi/4.1
 :jpa-spec: https://jakarta.ee/specifications/persistence/3.2/jakarta-persistence-spec-3.2
-
+[[a441]]
 === Jakarta Persistence & Jakarta Context and Dependency Injection (CDI) Integration
 This section specifies the requirements for Jakarta EE environments that integrate both {jpa-spec}[Persistence] and {cdi-spec}[CDI].
 
+Several of the items described in this section are also available with the following additional access methods.
+
+- the _@Resource_ annotation with the JNDI _lookup_ attribute.
+- the JNDI lookup using JNDI API.
+
+The items available with these additional access methods are indicated inline.
```

Comment:
Could we get a reference to sections 5.13 and 5.14 here, please? Since those sections specify how the JNDI name is determined, I believe.